src/CachePlugin.php

```diff
@@ -33,7 +34,7 @@ final class CachePlugin implements Plugin
     private $pool;
 
     /**
-     * @var StreamFactory
+     * @var StreamFactory|StreamFactoryInterface
```

Comment:
once https://github.com/php-http/message-factory/pull/38 is released, we can raise our version constraint to get that version and only use the psr interface from then on.

Reply:
Yes, and let ping me at this moment, I will be glad to do the PR ;)